projects/php/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "http://php.net/"
+primary_contact: "stas@php.net"
+auto_ccs:
+ - "smalyshev@gmail.com"
+experimental: True
```

Comment:
I think we can remove this and start without experimental mode :) Let me test it locally real quick.

Reply:
I'll merge the PR and remove this + disable MSan in a follow-up.